apps/andi/lib/andi_web.ex

```diff
@@ -66,6 +66,16 @@ defmodule AndiWeb do
     end
   end
 
+  def live_view do
+    quote do
+      use Phoenix.LiveView
+      import AndiWeb.ErrorHelpers
+      import Phoenix.HTML.Form
```

Comment:
I'm not sure that I'd import 'Form' and 'Link' since they won't be used in all live_view components

Reply:
These will generally be used in all components.

This is basically in line with what you would import in a non-live Phoenix View